gtsfm/utils/metrics.py

```diff
@@ -193,3 +184,19 @@ def get_rotations_translations_from_poses(
     metrics["translation_averaging_distance"] = compute_translation_distance_metrics(wti_aligned_list, gt_wti_list)
     metrics["translation_to_direction_angle_deg"] = compute_translation_angle_metrics(i2Ui1_dict, wTi_aligned_list)
     return metrics
+
+
```

Comment:
moved this callback to be an outright function, since i'll use it elsewhere in a future PR